--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,7 +1,13 @@
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
-    extractLibs "com.facebook.fbjni:fbjni:+:headers"
-    extractLibs "com.facebook.fbjni:fbjni:+"
-  }
+  def reactNativeVersionParts = REACT_NATIVE_VERSION.split("\\.")
+  def reactNativeMajor = reactNativeVersionParts[0].toInteger()
+  def reactNativeMinor = reactNativeVersionParts[1].toInteger()
+  if (reactNativeMajor == 0 && reactNativeMinor < 71) {
+    extractLibs "com.facebook.fbjni:fbjni:+:headers"
+    extractLibs "com.facebook.fbjni:fbjni:+"
+  }
