diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PhotoContainerFormat+fromImageFormat.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PhotoContainerFormat+fromImageFormat.kt
index ab32c97..9e5fc66 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PhotoContainerFormat+fromImageFormat.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PhotoContainerFormat+fromImageFormat.kt
@@ -7,7 +7,8 @@ import com.margelo.nitro.camera.utils.ImageFormatUtils
 fun PhotoContainerFormat.Companion.fromImageFormat(imageFormat: Int): PhotoContainerFormat {
   return when {
     // HEIC
-    imageFormat == ImageFormat.HEIC || imageFormat == ImageFormat.HEIC_ULTRAHDR -> PhotoContainerFormat.HEIC
+    // imageFormat == ImageFormat.HEIC || imageFormat == ImageFormat.HEIC_ULTRAHDR -> PhotoContainerFormat.HEIC
+    imageFormat == ImageFormat.HEIC -> PhotoContainerFormat.HEIC
     // DNG/RAW
     ImageFormatUtils.isRawFormat(imageFormat) -> PhotoContainerFormat.DNG
     // JPEG
diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PixelFormat+fromImageFormat.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PixelFormat+fromImageFormat.kt
index 2192b01..33fe564 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PixelFormat+fromImageFormat.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/extensions/converters/PixelFormat+fromImageFormat.kt
@@ -43,12 +43,12 @@ fun PixelFormat.Companion.fromImageFormat(
         else -> PixelFormat.YUV_420_10_BIT_FULL
       }
     }
-    ImageFormat.YCBCR_P210 -> {
-      when (range) {
-        PixelRange.VIDEO -> PixelFormat.YUV_422_10_BIT_VIDEO
-        else -> PixelFormat.YUV_422_10_BIT_FULL
-      }
-    }
+    // ImageFormat.YCBCR_P210 -> {
+    //   when (range) {
+    //     PixelRange.VIDEO -> PixelFormat.YUV_422_10_BIT_VIDEO
+    //     else -> PixelFormat.YUV_422_10_BIT_FULL
+    //   }
+    // }
     ImageFormat.PRIVATE -> PixelFormat.PRIVATE
     else -> PixelFormat.UNKNOWN
   }
diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/utils/ImageFormatUtils.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/utils/ImageFormatUtils.kt
index 11997f8..7272de7 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/utils/ImageFormatUtils.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/margelo/nitro/camera/utils/ImageFormatUtils.kt
@@ -36,11 +36,11 @@ object ImageFormatUtils {
       ImageFormat.DEPTH_JPEG,
       ImageFormat.RAW_SENSOR,
       ImageFormat.HEIC,
-      ImageFormat.HEIC_ULTRAHDR,
+      // ImageFormat.HEIC_ULTRAHDR,
     )
   private val allPhotoHdrFormats =
     arrayOf(
-      ImageFormat.HEIC_ULTRAHDR,
+      // ImageFormat.HEIC_ULTRAHDR,
       ImageFormat.JPEG_R,
     )
 
